buildifier/buildifier.go

```diff
@@ -105,7 +105,18 @@ func main() {
 	build.DisableRewrites = disable()
 	build.AllowSort = allowSort()
 
-	tables.FormatBzlFiles = *formatBzlFiles
+	// Check input type.
+	switch *inputType {
+	default:
```

Comment:
while this works, the convention is default is the last case and other than that looks wrong.

Reply:
I copied from the switch block below, should I fix it as well?

Reply:
Done.